README.md

```diff
@@ -8,3 +8,11 @@ Records are arranged alphabetically with authors, titles, and subjects interfile
 ## System dependencies
 
 This project uses [asdf](https://asdf-vm.com/) (see .tool-versions for the current ruby version)
+
+## How to start application locally 
+
+To run this application locally, we run the command to create a new rails application with the name `imagecat-rails`.
```

Comment:
Please remove the bit that says "create a new rails application..." 

I think all you need in this readme is "To run this application locally, run `bin/rails server`, then in a browser connect to localhost:3000"